cg/services/analysis_starter/configurator/implementations/mip_dna.py

```diff
@@ -26,18 +27,23 @@
 class MIPDNAConfigurator(Configurator):
     def __init__(
         self,
+        config: MipConfig,  # TODO: Consider a better name for this config
```

Comment:
Comment just to remember to fix this TODO. Resolve when we decide about the name